recipes-devtools/confuse/confuse_3.2.2.bb

```diff
@@ -3,12 +3,12 @@ HOMEPAGE = "http://www.nongnu.org/confuse/"
 LICENSE = "ISC"
 SECTION = "libs"
 
-LIC_FILES_CHKSUM = "file://src/confuse.c;beginline=1;endline=15;md5=90fe4e5c6740b817724d0da656d4155f"
+LIC_FILES_CHKSUM = "file://src/confuse.c;beginline=1;endline=15;md5=c6e0695034671ad80adcf767cdb505fd"
```

Comment:
Apparently, libconfuse now has a LICENSE file (since v2.8, see https://github.com/libconfuse/libconfuse/commit/f579c07dbc2308867a917551ed33e40b32f99490). Thus please refer to this instead.

Reply:
Thanks, don't know how I missed that. Fixed, and rebased resolving the conflict (keeping the added nativesdk of course).